src/pages/Report/Report.tsx

```diff
@@ -12,98 +13,94 @@ const Report: FC = () => {
   }
 
   return (
-    <div className="Reprt_1 bg-blue-100">
-      <div className="flex flex-col items-center mt-3">
-        <h1 className="text-2xl font-bold absolute top-30 left-0">Create New Report</h1>
-        <Link
-          href="/report_1"
-          underline="hover"
-          sx={{
-            position: 'absolute',
-            left: 0,
-            top: 50,
-            fontSize: 12,
-            fontWeight: 400,
-          }}
-        >
-          My Report
-        </Link>
-        <p
-          style={{
-            left: 60,
-            top: 48,
-            position: 'absolute',
-            color: 'rgba(3, 17, 27, 0.4)',
-          }}
-        >
-          &gt;&gt;
-        </p>
-        <h1
-          style={{
-            position: 'absolute',
-            fontSize: 12,
-            left: 80,
-            top: 50,
-          }}
-        >
-          Create New Report
-        </h1>
+    <div className="flex h-screen bg-blue-100">
+      <div className="w-1/5">    <ReportList /></div>
 
-        <div className="flex justify-end w-full">
-          <button type="button" className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">
-            Back to My Reports
-          </button>
-        </div>
+    <div className="flex-1">
+    <div className=" Reprt_1 h-screen bg-blue-100 flex flex-col mt-3 ">
+<div className="flex items-center ">
```

Comment:
use flex justify-between here, you can have the two children div seperate on the sides. 

then you can remove the flex on the children divs